### PR TITLE
Swift 5.0

### PR DESCRIPTION
**Pull request by devcarlos**
Swift 5.0 Upgrade

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ pod "Skeleton"
 Skeleton is also available through [Carthage](https://github.com/Carthage/Carthage). Add this to your Cartfile:
 
 ```
-github "gonzalonunez/Skeleton" ~> 0.3.1
+github "gonzalonunez/Skeleton" ~> 0.4.0
```
Comment (gonzalonunez) [inline README.md]: Perfect! 👍

Comment (gonzalonunez) [inline README.md]: Thank you for updating the README 🙏

File: Skeleton.podspec [hunk 2]
```diff
@@ -1,22 +1,8 @@
...
-# valid spec before submitting.
-#
-# Any lines starting with a # are optional, but their use is encouraged
-# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html
-#
-
 Pod::Spec.new do |s|
   s.name             = 'Skeleton'
-  s.version          = '0.3.1'
+  s.version          = '0.4.0'
```
Comment (gonzalonunez) [inline Skeleton.podspec:3]: Perfect 👍

File: Skeleton/CAGradientLayer+Slide.swift [hunk 3]
```diff
@@ -17,39 +17,40 @@ extension CABasicAnimation {
...
   
-  /** Slide a `CAGradientLayer` in a particular direction.
-   
-   - Important: If using `group`, modify the `CAAnimationGroup` responsibly. Setting the `CAAnimationGroup`s `animations` here **will overwrite the slide animations**.
-   
-   - parameter dir: The `Direction` to slide the `CAGradientLayer` in.
-   
-   - parameter group: A function that takes in and returns a `CAAnimationGroup`. Useful to modify the `CAAnimationGroup` that is used to animate the `CAGradientLayer`. By default, no modifications are made to the corresponding `CAAnimationGroup`.
-  */
-  public func slide(to dir: Direction, group: ((CAAnimationGroup) -> Void) = { _ in }) {
```
Comment (gonzalonunez) [inline Skeleton/CAGradientLayer+Slide.swift:31]: Can we go back to the original indentation settings? I believe it was two spaces but I'm not entirely sure.

**Review by gonzalonunez — COMMENTED** [review]
This looks great! One question about the indentation settings and we're set 🙂

File: Skeleton/CAGradientLayer+Slide.swift (hunk 3 above)
Comment (devcarlos) [reply Skeleton/CAGradientLayer+Slide.swift:31]: I have thought it was a mistake or something, so I have used the Xcode 11 automatic indentation with tabs, It is really required to get back ? seems tabs are the most common indentation these days, but I can go back for sure if you want 😄

**Comment by gonzalonunez** [comment]
I think it’s worth it! It would make the PR a bit cleaner :)

It’s Xcode’s tab indentation but with tab == 2 spaces.

**Comment by devcarlos** [comment]
Done 👍

File: Skeleton/Direction.swift [hunk 4]
```diff
@@ -32,22 +32,22 @@ public enum Direction {
...
+   s        e
+   ▄▄▄▄▄▄▄▄▄▄
    |----------|xxxxxxxxxx|----------|
    -1         0          1          2
    
-      ---->  ---->   ---->   ---->
+   ---->  ---->   ---->   ---->
    
-               s        e
-               ▄▄▄▄▄▄▄▄▄▄
```
Comment (gonzalonunez) [inline Skeleton/Direction.swift]: Can we undo all changes in this file?

**Review by gonzalonunez — COMMENTED** [review]
Almost there! Let's undo the changes to that comment and we'll be ready to merge 🙂

File: Skeleton/Direction.swift (hunk 4 above)
Comment (devcarlos) [reply Skeleton/Direction.swift]: Sure 👍

**Comment by devcarlos** [comment]
Restored and cleanup !

**Comment by devcarlos** [comment]
Passing Travis on my fork:

Screen Shot 2019-12-14 at 20 45 59

File: .travis.yml [hunk 5]
```diff
@@ -1,14 +1,17 @@
...
+podfile: Example/Podfile
+before_install:
+- gem install cocoapods # Since Travis is not always on latest version
+- pod setup
+- pod install --project-directory=Example
+xcode_workspace: Example/Skeleton.xcworkspace
+xcode_scheme: Skeleton-Example
+xcode_destination: platform=iOS Simulator,OS=13.0,name=iPhone 11 Pro
 script:
-- set -o pipefail && xcodebuild test -workspace Example/Skeleton.xcworkspace -scheme Skeleton-Example -sdk iphonesimulator9.3 ONLY_ACTIVE_ARCH=NO | xcpretty
```
Comment (gonzalonunez) [inline .travis.yml:13]: So much cleaner! Thank you 🙏

**Review by gonzalonunez — APPROVED** [review]
Thank you Carlos! Really appreciate your patience. Thanks again for contributing!! 🙂

**Comment by devcarlos** [comment]
Your welcome  😄👍 
PS. Don't forget to merge 🤓

**Comment by gonzalonunez** [comment]
Oh! I was thinking you’d be able to merge haha do you have access?

If not, I’ll merge tomorrow when I have WiFi again 🙂

**Comment by devcarlos** [comment]
It seems I don't have write permissions so you will have to merge this one 😅

**Comment by devcarlos** [comment]
Hey @gonzalonunez don't forget to increase the version tag and update Cocoapods

**Comment by gonzalonunez** [comment]
@devcarlos Ah! Thank you. On it – sorry, it's been a while 🤦‍♂

**Comment by gonzalonunez** [comment]
Just a heads up, I'm unable to update Cocoapods because the only computer I have access to can't run Xcode 11. I'll be able to update it on January 7th. So sorry about this 😣

**Comment by gonzalonunez** [comment]
@devcarlos Listo :)

**Pull request merged**